clients/fides-js/src/lib/consent-types.ts

```diff
@@ -605,6 +608,14 @@ export type OverrideOptions = {
   fides_embed: boolean;
   fides_tcf_gdpr_applies: boolean;
   fides_locale: string;
+  fides_primary_color: string;
+};
+
+export type OverrideExperienceTranslations = {
+  fides_title: string;
+  fides_description: string;
+  fides_privacy_policy_url: string;
+  fides_override_language: string;
 };
```

Comment:
I don't think I fully grok how these get set. Is it like this?
```
window.fides_overrides = {
  fides_title: "my custom title",
  fides_description: "my custom description",
  fides_privacy_policy_url: "https://example.com/my/custom/url",
  fides_override_language: "en"
}
```

If so I'm finding it confusing that `fides_title` wouldn't just always work, and it needs to be combined with the other `fides_override_language` option. Writing the docs for this might help us find a way that's easier to intuit

Reply:
right, it could be set via all 3 different ways we set our other overrides, including the `window.fides_overrides`, but also using cookie or query param.

RE: `I'm finding it confusing that fides_title wouldn't just always work`- I agree, and this was a compromise I had to make with keeping the implementation as generic as possible for overrides while supporting all the ways we set overrides. I'll add this to docs in a separate ticket- https://ethyca.atlassian.net/browse/PROD-1887